### PR TITLE
Alon cellular (YouPhone) has a new prefix: 9725587

### PR DESCRIPTION
**Pull request by nirberman**

### CODE REVIEW

**Comment by jias0001** [comment]
Thanks for the patch. Could you provide supporting documents for this change? Also, you need to update the PhoneNumberMetadata.xml to include the new prefix as well.

**Comment by nirberman** [comment]
As for the documentation: the new prefixes can be found in the Hebrew wiki page: http://he.wikipedia.org/wiki/%D7%A7%D7%99%D7%93%D7%95%D7%9E%D7%AA_%D7%98%D7%9C%D7%A4%D7%95%D7%9F_%D7%91%D7%99%D7%A9%D7%A8%D7%90%D7%9C
(look for 055-87X-XXXX). It seems the English page is not yet update with the new prefix.

I can also provide the contract which lists the phone number of the SIM card I got. Where shuld I email it?

As for PhoneNumberMetadata.xml, I updated it as well, it should be part of the PR.

File: resources/PhoneNumberMetadata.xml [hunk 1]
```diff
@@ -10015,7 +10015,7 @@
               5[5689]|
               6[67]|
               7[0178]|
-              8[89]|
+              8[7-9]|
```
Comment (jias0001) [inline resources/PhoneNumberMetadata.xml]: merge this with the next line to [89][7-9]. It is faster to process.

**Comment by jias0001** [comment]
No need to email me the contract - we will take your word for it :-) It would be great if you can help update the English wikipedia page as well.

File: resources/PhoneNumberMetadata.xml (hunk 1 above)
Comment (nirberman) [inline resources/PhoneNumberMetadata.xml]: Done.
Updated English wikipedia page as well.

**Comment by jias0001** [comment]
Could you make sure you sign the CLA?

https://github.com/googlei18n/libphonenumber/wiki/ContributeToTheProject

**Comment by nirberman** [comment]
Signed.


Regards,
Nir Berman
CTO, co-founder
+972 (52) 663 6920
nir@simgo-mobile.com

**Pull request merged**